loony_dev/tests/test_web.py

```
@@ -2163,5 +2177,162 @@ def test_snapshot_failure_falls_back_to_empty_not_500(self) -> None:
         self.assertEqual(resp.json(), [])
 
 
+class _FakeIssueRecorder:
+    """An ``Issue`` stand-in recording add_label / remove_label calls (#225)."""
+
+    instances: list["_FakeIssueRecorder"] = []
```

Ruff (0.15.17)

[warning] Mutable default value for class attribute

(RUF012)